Sources/Quick/Filter.swift

```diff
@@ -22,9 +22,17 @@ final public class Filter: NSObject {
     }
 
     /**
-        Example and example groups with [Pending: true] are excluded from test runs.
+        Example and example groups with [Excluded: true] are excluded from test runs.
         Use this to temporarily suspend examples that you know do not pass yet.
     */
```

Comment:
nit-pick: This documentation applies more to "pending" examples than "excluded" ones. I'd rephrase slightly:

```
Example and example groups with [Excluded: true] are not included in test runs.
Use this to disable examples that do not pass. Of course, you could also comment
the examples out altogether, but excluding them will ensure they continue to at
least compile, which allows them to be re-enabled more easily in the future.

In addition, future versions of Quick may be able to point out when a disabled test
passes, and thus should be re-enabled.
```